beacon/endpoints/rest/response/response_schema.py

```diff
@@ -1,48 +1,53 @@
 import logging
 
-import itertools
-
 from .... import conf
-from ..schemas import SUPPORTED_SCHEMAS, DEFAULT_SCHEMAS
 from ....utils.json import jsonb
 
 LOG = logging.getLogger(__name__)
 
 
-def build_beacon_response(data, qparams_converted, func_response_type, variant_id=None, individual_id=None,
-                          biosample_id=None,):
+def build_beacon_response(data,
+                          qparams_converted,
+                          non_accessible_datasets,
+                          func_response_type,
+                          variant_id=None,
+                          individual_id=None,
+                          biosample_id=None):
     """"
     Transform data into the Beacon response format.
     """
 
     beacon_response = {
-        'meta': build_meta(qparams_converted, func_response_type, variant_id, individual_id, biosample_id),
-        'response': build_response(data, qparams_converted, func_response_type)
+        'meta': build_meta(qparams_converted, variant_id, individual_id, biosample_id),
+        'response': build_response(data, qparams_converted, non_accessible_datasets, func_response_type)
     }
     return beacon_response
 
 
-def build_meta(qparams, func_response_type, variant_id=None, individual_id=None, biosample_id=None):
+def build_meta(qparams, variant_id=None, individual_id=None, biosample_id=None):
     """"Builds the `meta` part of the response
 
     We assume that receivedRequest is the evaluated request (qparams) sent by the user.
     """
 
+    schemas = [qparams.requestedAnnotationSchema[0]] if hasattr(qparams, 'requestedAnnotationSchema') else []
```

Comment:
Better to go through the qparams and get the "SchemaField" object and print their names.
Sometimes you'll get one, sometimes you'll get more.
Better it'll be independant from how the field is called.

Reply:
How do I do this?